exercises/tasks/task_07.py

```diff
@@ -25,6 +27,16 @@ def test_sorting_panel_actions_can_be_used_for_sorting_the_search_results(page):
     # 3. Select the Cheapest sorting option from the sorting panel
 
     # 4. Store the price value of the first few (e.g., 5) results into a (Python) list
```

Comment:
I'd leave this for the attendees to figure out, but perhaps it really is too obscure without any hints 🥲